integration/container/cdi_test.go

```diff
@@ -72,55 +76,49 @@ func TestCDISpecDirsAreInSystemInfo(t *testing.T) {
 	testCases := []struct {
 		description             string
 		config                  map[string]interface{}
-		experimental            bool
 		specDirs                []string
 		expectedInfoCDISpecDirs []string
 	}{
 		{
-			description:             "experimental no spec dirs specified returns default",
-			experimental:            true,
+			description:             "CDI enabled with no spec dirs specified returns default",
```

Comment:
Very unrelated to this PR, but perhaps something to look into;

During tests, we use paths based on the test-name. If I use a _comma_ in the test name, for example;

```suggestion
			description:             "CDI enabled, with no spec dirs specified returns default",
```

Then the test-daemon fails to start;

```
    cdi_test.go:148: [d9c505ae70399] failed to start daemon with arguments [--data-root /go/src/github.com/docker/docker/bundles/test-integration/TestCDISpecDirsAreInSystemInfo/CDI_enabled,_with_no_spec_dirs_specified_returns_default/d9c505ae70399/root --exec-root /tmp/dxr/d9c505ae70399 --pidfile /go/src/github.com/docker/docker/bundles/test-integration/TestCDISpecDirsAreInSystemInfo/CDI_enabled,_with_no_spec_dirs_specified_returns_default/d9c505ae70399/docker.pid --userland-proxy=true --containerd-namespace d9c505ae70399 --containerd-plugins-namespace d9c505ae70399p --containerd /var/run/docker/containerd/containerd.sock --host unix:///tmp/docker-integration/d9c505ae70399.sock --debug --storage-driver vfs --config-file=/tmp/TestCDISpecDirsAreInSystemInfoCDI_enabled,_with_no_spec_dirs_specified_returns_default20488269/001/daemon.json] : [d9c505ae70399] daemon exited during startup: exit status 1
    --- FAIL: TestCDISpecDirsAreInSystemInfo/CDI_enabled,_with_no_spec_dirs_specified_returns_default (0.51s)
```

And looking in the daemon logs it appears that boltdb doesn't like such paths (possibly considering the comma to be a comma-separated list of paths?);

```
failed to start daemon: Error initializing network controller: error obtaining controller instance: boltdb supports one endpoint and should be a file path
```